crypto/packet.c

```diff
@@ -9,6 +9,9 @@
 
 #include "internal/cryptlib.h"
 #include "internal/packet.h"
+#ifndef FIPS_MODULE
+# include "internal/packet_quic.h"
+#endif
```

Comment:
Wouldn't it be cleaner to have a separate packet_quic.c?

Reply:
I tried that but I've realized it is not possible given the put_quic_value() call in wpacket_intern_close().